src/general/NewGameSettings.cs

```diff
@@ -0,0 +1,676 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using Godot;
+
+public class NewGameSettings : ControlWithInput
+{
+    [Export]
+    public NodePath BasicOptionsPath = null!;
+
+    [Export]
+    public NodePath AdvancedOptionsPath = null!;
+
+    [Export]
+    public NodePath BasicButtonPath = null!;
+
+    [Export]
+    public NodePath AdvancedButtonPath = null!;
+
+    [Export]
+    public NodePath TabButtonsPath = null!;
+
+    [Export]
+    public NodePath DifficultyTabPath = null!;
+
+    [Export]
+    public NodePath PlanetTabPath = null!;
+
+    [Export]
+    public NodePath MiscTabPath = null!;
+
+    [Export]
+    public NodePath DifficultyTabButtonPath = null!;
+
+    [Export]
+    public NodePath PlanetTabButtonPath = null!;
+
+    [Export]
+    public NodePath MiscTabButtonPath = null!;
+
+    [Export]
+    public NodePath DifficultyPresetButtonPath = null!;
+
+    [Export]
+    public NodePath DifficultyPresetAdvancedButtonPath = null!;
+
+    [Export]
+    public NodePath MPMultiplierPath = null!;
+
+    [Export]
+    public NodePath MPMultiplierReadoutPath = null!;
+
+    [Export]
+    public NodePath MutationRatePath = null!;
+
+    [Export]
+    public NodePath MutationRateReadoutPath = null!;
+
+    [Export]
+    public NodePath CompoundDensityPath = null!;
+
+    [Export]
+    public NodePath CompoundDensityReadoutPath = null!;
+
+    [Export]
+    public NodePath PlayerDeathPopulationPenaltyPath = null!;
+
+    [Export]
+    public NodePath PlayerDeathPopulationPenaltyReadoutPath = null!;
+
+    [Export]
+    public NodePath GlucoseDecayRatePath = null!;
+
+    [Export]
+    public NodePath GlucoseDecayRateReadoutPath = null!;
+
+    [Export]
+    public NodePath OsmoregulationMultiplierPath = null!;
+
+    [Export]
+    public NodePath OsmoregulationMultiplierReadoutPath = null!;
+
+    [Export]
+    public NodePath FreeGlucoseCloudButtonPath = null!;
+
+    [Export]
+    public NodePath MapTypeButtonPath = null!;
+
+    [Export]
+    public NodePath LifeOriginButtonPath = null!;
+
+    [Export]
+    public NodePath LifeOriginButtonAdvancedPath = null!;
+
+    [Export]
+    public NodePath LAWKButtonPath = null!;
+
+    [Export]
+    public NodePath LAWKAdvancedButtonPath = null!;
+
+    [Export]
+    public NodePath GameSeedPath = null!;
+
+    [Export]
+    public NodePath GameSeedAdvancedPath = null!;
+
+    [Export]
+    public NodePath IncludeMulticellularButtonPath = null!;
+
+    [Export]
+    public NodePath EasterEggsButtonPath = null!;
+
+    [Export]
+    public NodePath ConfirmButtonPath = null!;
+
+    private PanelContainer basicOptions = null!;
```

Comment:
This is a bit long one long block of variables. Usually our scenes have "logical" grouping (for example based on the subtab) for variables like this to make it easier to read and find related variables.